DATA_FORMAT.md

```diff
@@ -0,0 +1,77 @@
+# JSON Data fields
+
+## Message Data
+These fields are the primary data fields containing the most basic message data and used to identify the specific device. 
+For some devices these are the *only* fields contained in the message, as the message itself consitutes an event from this
+particular device model.
+
+* **time** (string) (Required)
+  * Time stamp. String containing date and time of when the message was received. Format is dependent on
+    current locale unless the `-U` command line argument is used.
+
+* **model** (string) (Required)
+  * Device model. Human readable string describing the model consisting of manufacturer name 
+    and manufacturers model description
+
+* **id** (string) (Optional)
+  * Device identification. Used to differentiate between devices of same *model*. 
+    Depending of device model it may be a non-volatile value programmed into the device, 
+    a volatile value that changes at each power on (or battery change), or a value configurable by
+    user e.g. by switch or jumpers. No assumptions should be made to the id value other than it contains
+    a unique sequence of alphanumeric characters.
+
+* **channel** (string) (Optional)
+  * Secondary device identification. For devices with more than one identification value 
+    (e.g. both an internal value and a switch).
+
+* **mic** (string) (Optional)
+  * Message integrity check. String describing the method used for ensuring the data integrity
+    of the message. Protocol decoders for devices without mic will be disabled by default as 
+    they are prone to excessive false positives.
+  * Possible values:
+    * "CRC" - Cyclic Redundancy Check.
+    * "CHECKSUM" - Accumulated sum of data.
+    * "PARITY" - Parity bit (odd, even, multiple)
+
+## Common Device Data
+Various data fields, which are common across devices of different types 
+
+* **battery** (string) (Optional)
```

Comment:
This is a case where if we are writing a specification we should attempt to set a standard we want everything to follow.

I propose 
- `battery` (with no units) should be considered deprecated. 
- `battery_low` or `battery_ok` should be added as a boolean (or an int is essentially a boolean).

Reply:
This is a hot topic. The idea is clear, "low battery" is an "event" thus "1" anything else is default and disregarded, hence "0".
The opposite reasoning is:
- devices that report a voltage or known steps should use `battery_mV`
- other devices should report a level between 0 (empty) and 1 (full)
- if only a flag is available this degrades to quasi-bool 0/1